core/trino-main/src/test/resources/cert/generate.sh

```diff
@@ -2,6 +2,16 @@
 
 set -eux
 
-openssl req -new -x509 -newkey rsa:4096 -sha256 -nodes -keyout localhost.key -days 3560 -out localhost.crt -config localhost.conf
```

Comment:
I don't see pem file changed.

Reply:
`cat "$tempdir/localhost.crt" "$tempdir/localhost.key" > "$outdir/localhost.pem"`

Reply:
@kokosing you mean the one committed in the repo? It doesn't need to be updated.

@lukasz-walkiewicz sorry but I don't get what you mean by that suggestion.